packages/discovery-styles/scss/components/document-preview/_pdfjs_web_mixins.scss

```diff
@@ -0,0 +1,83 @@
+@mixin pdfjsTextLayer {
```

Comment:
So this is the file which is generated from `update-styles.sh`? If so, please add a comment stating as such, so that future maintainers know not to manually edit/update. Something like:

```css
/* DO NOT EDIT. THIS FILE IS AUTOMATICALLY GENERATED FROM `update-styles.sh`. */
```